src/main/resources/content/k8s/containers-must-be-run-as-non-root.json

```diff
@@ -1 +1,8 @@
-{"severity":"high","rule":"$.spec.template.spec.containers[*].securityContext.runAsNonRoot exists and is false","id":"2e22737c-a5b8-4808-8a8b-d99fc7e99505","enabled":true,"resourceType":"k8s","policy":"containers must be run as non-root"}
\ No newline at end of file
+{
+  "severity": "high",
+  "rule": "$.spec.template.spec.containers[*].securityContext.runAsNonRoot exists and is false",
+  "id": "2e22737c-a5b8-4808-8a8b-d99fc7e99505",
+  "enabled": true,
+  "resourceType": "k8s",
+  "policy": "containers must be run as non-root"
```

Comment:
description field missing here as well.
Also, the policy name is not sentence case.